policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/solidarity/amount/qst/base.yaml

```diff
@@ -0,0 +1,15 @@
+description: Quebec provides this Quebec sales tax credit component of its Solidarity Tax Credit.
```

Comment:
capitalize solidarity tax credit consistently

@PavelMakarchuk please advise on the standard youve adopted

Reply:
I changed to lowercase to be the same as other programs